### DISCUSSION TITLE
What is the tallest waterfall in the 🇺🇸?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#900] [Galaxy Trivia] [answered]
**Bonus:** What is the tallest waterfall in the 🗺️?

**Comment by peterbaccaro** [comment]
The tallest waterfall in the United States is **Yosemite Falls**, located in Yosemite National Park in California. Yosemite Falls has a total height of 2,425 feet (739 meters) and consists of three main sections: Upper Yosemite Fall (1,430 feet or 436 meters), Middle Cascades (675 feet or 206 meters), and Lower Yosemite Fall (320 feet or 98 meters). It is one of the most iconic natural attractions in the park and attracts numerous visitors each year.

The tallest waterfall in the world is **Angel Falls**, located in Canaima National Park in Venezuela. Angel Falls has a total height of 3,212 feet (979 meters) and a continuous free fall of 2,648 feet (807 meters). It is named after Jimmy Angel, an American aviator, who was the first to fly over the falls in 1933. Angel Falls is a magnificent natural wonder and a popular tourist destination in Venezuela.

**Reply by ParanoidUser** [reply]
Yosemite holds the #6 spot.

**Reply by peterbaccaro** [reply, marked as answer]
OK, the tallest waterfall in the United States is **Olo'upena Falls** located in Hawaii which has a height of 2,953 feet or 900 metres.

https://www.worldatlas.com/falls/the-10-tallest-waterfalls-in-the-united-states.html
